base/Dockerfile.go

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.13.8-alpine
+
+LABEL maintainer="core-tech@better.com"
+
+COPY scripts/common.sh common.sh
+RUN apk update && apk upgrade && chmod +x common.sh && ./common.sh
```

Comment:
This now needs to look like the similar sequences from `base/Dockerfile.node` and `base/Dockerfile.python`:

```
ARG BASE_APK_DEPENDENCIES

# Install the base requirements
COPY                                         \
  scripts/common/install-rds-certificates.sh \
  scripts/common/install-sops.sh             \
  /tmp/scripts/common/
RUN :                                                         \
  && apk update                                               \
  && apk upgrade                                              \
  && apk add --no-cache                                       \
    # Base dependencies                                       \
    ${BASE_APK_DEPENDENCIES}                                  \
  # Run common install scripts                                \
  && chmod +x /tmp/scripts/common/install-rds-certificates.sh \
  && chmod +x /tmp/scripts/common/install-sops.sh             \
  && /tmp/scripts/common/install-rds-certificates.sh          \
  && /tmp/scripts/common/install-sops.sh                      \
  ;
```